src/announcer/announcer.js

```diff
@@ -124,8 +127,22 @@ const processQueue = async () => {
     debounce = setTimeout(() => {
       Log.debug(`Announcer - speaking: "${message}" (id: ${id})`)
 
+      // Merge global defaults with per-call options
+      const mergedOptions = { ...globalDefaultOptions, ...options }
+
+      // Extract utterance options from merged options object
+      const utteranceOptions = {
+        message,
+        id,
+        lang: mergedOptions.lang,
+        pitch: mergedOptions.pitch,
+        rate: mergedOptions.rate,
+        voice: mergedOptions.voice,
+        volume: mergedOptions.volume,
+      }
+
```

Comment:
we could probably do this is a single spread? that would save us from creating and allocating 2 variables over one.

If it still reads well, we could also consider doing it inline in the `speak` function on L145